cron/centreon-send-stats.php

```diff
@@ -102,6 +102,11 @@ function logger(string $message, Throwable $exception = null)
     }
 }
 
+// Don't send telemetry & statistics if the Open Source Edition administrator has made this choice
+if (! $shouldSendStatistics && ! $hasValidLicenses) {
+    exit;
+}
+
 // Only send telemetry & statistics if it's a Centreon central server
 if ($isRemote === false) {
```

Comment:
These checks can be added here instead

Reply:
Thank you @sc979 for your review, I just updated the code accordingly 👍